notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "getMessage"
+      summary: "Get Message"
+      tags: 
+        - "Messages"
+      description: "Retrieve a message already submitted for delivery."
+      responses: 
+        200: 
+          description: "Message found."
+          schema: 
+            $ref: "#/definitions/Message"
+  /Users/{id}/Messages: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
```

Comment:
I was thinking of oauth flow (and/or some "/User/Me" endpoint used to retrieve the current user's data without having to know the uuid in advance). But I agree it's an edge case and we can use the CF instead. This is one of the changes that assumes we are diverging from the SCIM specs. I'm still not convinced this is the more convenient path but let's start with this piece :)